backend/game_engine/firestore.py

```diff
@@ -414,3 +415,28 @@ def clear_votes(self) -> None:
         votes = self._client.collection('votes').stream()
         with ThreadPoolExecutor(max_workers=self._thread_pool_size) as executor:
             executor.submit(self._delete_vote_fn, votes)
+
+    def find_matchmaker_games(self, region="US", time_now=datetime.datetime.now()) -> list:
+        # Time comparison is done solely based on the hour
+        games_list = []
+        db = self._client
+        games = db.collection(u'games').stream()
+
+        for game in games:
+            try:
+                country = game.get("country_code")
+
+                start = game.get("game_start_time")
+
+                start_time = datetime.datetime.strptime(time_now.strftime('%Y/%m/%d ') + start, '%Y/%m/%d %H:%M')
```

Comment:
We may not need a game_start_time field. Instead, each game document could have a boolean field 'game_has_started' or equivalent. The matchmaker would then query all non-started games, and start them if eligible.